### PR TITLE
Cargo: Use require() to shorten code

### PR DESCRIPTION
**Pull request by sschuberth**
And improve the failure message along the way.

Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch-si.com>

### CODE REVIEW

File: analyzer/src/main/kotlin/managers/Cargo.kt [hunk 1]
```diff
@@ -138,8 +138,9 @@ class Cargo(
     private fun resolveLockfile(metadata: JsonNode): File {
         val workspaceRoot = metadata["workspace_root"].textValueOrEmpty()
         val lockfile = File(workspaceRoot, "Cargo.lock")
-        if (!lockfile.isFile) {
-            throw IllegalArgumentException("missing Cargo.lock file")
+
+        require(lockfile.isFile) {
```
Comment (fviernau) [inline analyzer/src/main/kotlin/managers/Cargo.kt]: out of scope: Should this adhere to allowDynamic version as well? e.g. be moved into the caller?

Comment (mnonnenmacher) [reply analyzer/src/main/kotlin/managers/Cargo.kt]: I wanted to refactor this anyway after #1789 is merged, to align it with how it is implemented in the other package managers.

Comment (sschuberth) [reply analyzer/src/main/kotlin/managers/Cargo.kt]: Great, so I'll merge this as-is and leave the rest to you 😄

Comment (mnonnenmacher) [reply analyzer/src/main/kotlin/managers/Cargo.kt]: The alignment PR is ready: #1794

**Pull request merged**